content/accessibility/designer-toolkit.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Designer toolkit
+---
+
+Whether you’re designing a new user experience or just one component, make sure you’re addressing the considerations from the respective list.
+## Accessibility checklists
+### Component creation
+Check out a11yportal's [custom component creation checklist for UX designers](https://www.a11yportal.com/guidelines/fundamentals/checklists.html#custom-control-checklist-ux).
```

Comment:
What does component creation mean?

Reply:
The links takes you to custom control checklist, is that right? Why is this relevant? (not questioning, but feel like it needs a sentence here that answers this question)

Reply:
Component creation refers to creating an interactive control that isn't using built-in semantic markup.

I can add in some explanation there, and change component to control, like how it is in the checklist I link out to